lib/api/sfApi.js

```diff
@@ -214,34 +78,56 @@ async function createReconciliationText(
   }
 }
 
-async function fetchReconciliationTexts(firmId, page = 1, refreshToken = true) {
-  setAxiosDefaults(firmId);
+async function readReconciliationTexts(firmId, page = 1, refreshToken = true) {
+  apiUtils.setAxiosDefaults(firmId);
   try {
     const response = await axios.get(`reconciliations`, {
       params: { page: page, per_page: 2000 },
     });
-    responseSuccessHandler(response);
-    return response;
+    apiUtils.responseSuccessHandler(response);
+    return response.data;
   } catch (error) {
     const callbackParameters = {
       firmId: firmId,
       page: page,
       refreshToken: false,
     };
-    const response = await responseErrorHandler(
+    const response = await apiUtils.responseErrorHandler(
       firmId,
       error,
       refreshToken,
-      fetchReconciliationTexts,
+      readReconciliationTexts,
       callbackParameters
     );
     return response;
   }
 }
 
-async function findReconciliationText(firmId, handle, page = 1) {
-  const response = await fetchReconciliationTexts(firmId, page);
-  const reconciliations = response.data;
+async function readReconciliationTextById(firmId, id, refreshToken = true) {
+  apiUtils.setAxiosDefaults(firmId);
+  try {
+    const response = await axios.get(`reconciliation/${id}`);
+    apiUtils.responseSuccessHandler(response);
+    return response;
+  } catch (error) {
+    const callbackParameters = {
+      firmId: firmId,
```

Comment:
No need to define property a property value if the name of the value & property match, so only `firmId` would work here too.